src/framework/shortcuts/data/shortcuts-Mac.xml

```diff
@@ -841,12 +833,48 @@
     <seq>P</seq>
     </SC>
   <SC>
-    <key>next-score</key>
+    <key>notation-next</key>
     <std>20</std>
     </SC>
   <SC>
-    <key>previous-score</key>
-    <std>21</std>
+    <key>notation-previous</key>
+    <seq>Ctrl+Shift+Tab</seq>
+    </SC>
```

Comment:
One thing you couldn't know: on macOS, `Ctrl` maps to the Cmd key, and Cmd+Tab is a shortcut reserved by the OS (it does something similar to Alt+Tab on Windows). Instead, it is common for macOS apps to use Ctrl+Tab. To achieve that, you should write `Meta+Tab` in this `shortcuts-Mac.xml` file.

But actually, I think you should just use `<std>21</std>`. That will be interpreted as `QKeySequence::StandardKey::PreviousChild`, which maps to the desired shortcut automatically based on the platform.

Reply:
What about the Ctrl+Shift+#Number#. Should that still be Ctrl or Meta?